backend/main.go

```diff
@@ -231,17 +242,33 @@ func main() {
 	router := mux.NewRouter()
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/ws", hub.Handler))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/download/{path:.*}", nomad.downloadFile))
-	router.PathPrefix("/static").Handler(http.FileServer(myAssetFS))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/config.js", func(w http.ResponseWriter, r *http.Request) {
 		response := make([]string, 0)
 		response = append(response, fmt.Sprintf("window.NOMAD_READ_ONLY=%s", strconv.FormatBool(cfg.ReadOnly)))
 		response = append(response, fmt.Sprintf("window.NOMAD_ADDR=\"%s\"", cfg.Address))
 		response = append(response, fmt.Sprintf("window.NOMAD_LOG_LEVEL=\"%s\"", cfg.LogLevel))
 
+		var endpointURL *url.URL
+		var err error
+		if cfg.ProxyAddress != "" {
+			endpointURL, err = url.Parse(cfg.ProxyAddress)
+			if err != nil {
+				logger.Infof("Error parsing supplied web.proxy-address of %s: %s", cfg.ProxyAddress, err)
```

Comment:
the check should be moved to the configuration parser / validation, so we avoid this as a runtime error :)